docs/errors/validation_errors.md

```diff
@@ -372,7 +372,6 @@ This error is also raised for strict fields when the input value is not an insta
 
 This error is raised when the input value is a string that cannot be parsed for a `datetime` field:
 
-<!-- uncomment with new pydantic-core version
 ```py test="skip"
```

Comment:
I think? When I was putting this in, I think `pytest-examples` was still trying to run the test even though it was commented, but I can't remember.

```suggestion
```py
```